src/test/dataMigration/stepsPanEuropean/countryComments/migrateCountryComments.ts

```diff
@@ -64,6 +64,11 @@ export const migrateCountryComments = async (props: Props, client: BaseProtocol)
     const tableName = `country_comments_${tableIdx}`
     Logger.info(`==== ${i + 1}/${fileNames.length} (${Numbers.format(p)})%  -> ${fileName} `)
     const table = tables[tableName]
+    if (!table) {
```

Comment:
are these tables just hidden or not in the database ?
at line 38, when fetching metadata , pass the parameter to include hidden sections. data must be imported .